src/Microsoft.IdentityModel.Abstractions/DataClassification.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.IdentityModel.Abstractions
+{
+    /// <summary>
+    /// Data classification categories defined by the asset classification standard.
+    /// </summary>
+    /// <remarks>
+    /// See http://aka.ms/AssetClassificationStandard for more details about the asset classification standard.
```

Comment:
Updated to: https://aka.ms/EntOnlineSvcDataTaxonomy
Unsure if it's kosher to include a link one must authenticate to read? This will be akin to a dead link for those outside MSFT.

Reply:
decided to go a different direction